src/ngraph/frontend/onnx_import/op/reduce.hpp

```diff
@@ -128,8 +143,13 @@ namespace ngraph
                 ///
                 inline NodeVector reduce_max(const Node& node)
                 {
-                    return {reduction::make_ng_reduction_op<ngraph::op::Max>(
-                        node, node.get_ng_inputs().at(0))};
+                    return {reduction::make_ng_reduction_op(
+                        node,
+                        node.get_ng_inputs().at(0),
+                        [](const std::shared_ptr<ngraph::Node>& input,
+                           const ngraph::AxisSet& reduction_axes) {
+                            return std::make_shared<ngraph::op::Max>(input, reduction_axes);
```

Comment:
Could you just `std::make_shared<ngraph::op::Max>` in place of the lambda?

Reply:
Yup.